src/components/LoginPage/Slide4.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import slideImg1 from 'components/LoginPage/images/slide-4_img-1.svg';
+import slideImg2 from 'components/LoginPage/images/slide-4_img-2.svg';
+
+const Slide = styled.section`
+  padding-bottom: 10%;
+  text-align: center;
+  position: relative;
+
+<<<<<<< HEAD
+=======
+    .shadow {
+    -webkit-filter: drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3));
+    filter: drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3));
+    }
+>>>>>>> 0dfe1e73b0b826d4e2cba58197e330290b9fcca4
+    .img1 {
+        position: absolute;
+        width: 60%;
+        right: 0%;
+        top: 3%;
+        z-index: 1;
+    }
+    .img2 {
+        width: 100%;
+        margin-left: -24%;
+<<<<<<< HEAD
+        -webkit-filter: drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3));
+        filter: drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3));
+=======
+>>>>>>> 0dfe1e73b0b826d4e2cba58197e330290b9fcca4
+    }
+`;
+const Figure = styled.figure`
+  position: relative;
+  width: 74%;
+  margin: auto;
+  height: auto;
+`;
+const Img = styled.img`
+  width: 100%;
+  height: auto;
+`;
+const Caption = styled.svg`
+  font-weight: ${props => props.fontWeight || 300};
+  position: absolute;
+  width: ${props => props.width};
+  top: ${props => props.top};
+  left: ${props => props.left};
+  right: ${props => props.right};
+  bottom: ${props => props.bottom};
+`;
+
+const CaptionSvgTextBox = ({ viewBoxWidth = 36, viewBoxHeight = 22, ...rest }) => {
+    return <Caption viewBox={`0 0 ${viewBoxWidth} ${viewBoxHeight}`} {...rest} />;
+};
+const CaptionSvgText = ({ x = 0, y = 12, fill = "#96FFFF", text }) => {
```

Comment:
`CaptionSvgText ` is declared in every Slide file. Since it's a component that is shared between all the slides, might make sense to move it out to a separate file and import in every Slide file.